.travis.yml

```diff
@@ -49,15 +46,22 @@ cache:
     - .eslintcache
     - $HOME/.cache/electron-builder
 
-before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.5.1
-  - export PATH=$HOME/.yarn/bin:$PATH
+env:
+  - YARN_VERSION=1.9.2
+  - CC=clang
+  - CXX=clang++
+  - npm_config_clang=1
 
 install:
-  - yarn install --force
+  - node ./vendor/yarn-$YARN_VERSION.js install --force
 
 script:
-  - yarn lint && yarn validate-changelog && yarn build:prod && yarn test:setup && yarn test
+  - |
+    node ./vendor/yarn-$YARN_VERSION.js lint &&
```

Comment:
Does this version of Yarn work properly when subcommands (like this one) run `yarn`?

Reply:
This is what I'm digging into now. [`.yarnrc`](https://yarnpkg.com/lang/en/docs/yarnrc/) supports `local-path` if you want to override the global yarn with a local version, and Travis seems to be working fine.

<img width="813" src="https://user-images.githubusercontent.com/359239/43651392-9ab4c57e-9718-11e8-8616-8d1f454a9958.png">

I want to verify it across all the platforms, hence doing it one at a time.

Reply:
Actually, it doesn't seem to be currently. Sigh.

<img width="503" src="https://user-images.githubusercontent.com/359239/43651457-d36577e2-9718-11e8-9ad8-9eb31c4db47e.png">